torch/lib/ATen/dlpack.h

```diff
@@ -114,6 +114,19 @@ typedef struct {
   /*! \brief The offset in bytes to the beginning pointer to data */
   uint64_t byte_offset;
 } DLTensor;
+
+/*!
+ * \ manage memory of DLTensor
+ */
+typedef struct {
+  /*! DLTensor which is being memory managed*/
+  DLTensor dlTensor;
+  /* context in which DLTensor is used in framework*/
+  void * ctx;
+ /* Destructor signature */
```

Comment:
you need to describe whether destructor is supposed to delete the instance of DLManagedTensor or not. Alternatively it might not (and it's responsibility of caller) and then we can allow for destructor==nullptr

Reply:
Yes, we should allow for destructor to be nullptr in the case where there is API that requires a managed tensor but there is no way for the caller to actually provide a reasonable destructor.